test/mock/femr/common/MockItemModelMapper.java

```diff
@@ -145,4 +145,9 @@ public UserItem createUserItem(IUser user) {
     public VitalItem createVitalItem(String name, Float value) {
         return null;
     }
+
+    @Override
+    public InternetStatusItem createInternetStatusItem(IInternetStatus status) {
```

Comment:
Should we be testing this more thoroughly?

Reply:
This method was only to pass integration checks since it complained about a method not being implemented since the class implements IItemModelMapper, an interface that I previously changed by adding an internet status item. I saw that they were only testing two services and felt as if it wasn't a huge deal. Plus, I personally don't think we need to test for internet status but I could be wrong.